code/modules/mob/living/carbon/human/human.dm

```diff
@@ -384,17 +389,23 @@
 				if(pocket_item)
 					if(pocket_item == (pocket_id == slot_r_store ? r_store : l_store)) //item still in the pocket we search
 						unEquip(pocket_item)
+						if(has_pickpocket == 1)
+							var/mob/living/carbon/human/H = usr
+							if(H.hand) //left active hand
```

Comment:
I am 99% sure this isn,t how you do it,check ifthere is a proc to put something in an hand